internal/docker/supported.go

```diff
@@ -25,15 +25,25 @@ func startDbInDockerSupported(dialect string) (cleanup func() error, retURL, con
 	}
 
 	var resource *dockertest.Resource
-	var url string
+	var url, tag string
+
+	opts := GetOpts(opt...)
+	if opts.withContainerImage != "" {
+		dialect, tag, err = splitImage(opt...)
```

Comment:
I am a little confused with what is going on in this path, we parse the `withContainerImage` then call this function with the `dialect` as well as the the option.  Here we parse it again setting the `dialect` parameter we passed in?

Reply:
I think this makes more sense *now* (as of recent commit), but I'm still thinking about it.